getDeployConfig.js

```diff
@@ -8,25 +8,25 @@ const os = require('os')
 const projectRoot = require('app-root-path')
 const appPackage = require(projectRoot.path + '/package.json')
 const commandLineArguments = require('yargs-parser')(process.argv.slice(2), { alias: { deployFile: ['df'] } })
+const logger = require('roosevelt-logger')({
+  disable: ['LOADED_MOCHA_OPTS']
```

Comment:
why are we setting this option outside of the tests?

Reply:
So my mindset initially was to put the disable flag wherever we actually were writing the logs but after a discussion with @mulligan121, I switched it to running the following statement inside the globalHooks as part of the before all hook:

```js
requrire('roosevelt-logger').winstonInstance.silent = true
```